dev/builddeps-veloxbe.sh

```diff
@@ -32,6 +32,7 @@ VELOX_BRANCH=""
 VELOX_HOME=""
 VELOX_PARAMETER=""
 COMPILE_ARROW_JAVA=OFF
+NUM_THREADS=$(nproc --ignore=2)
```

Comment:
Maybe, someone prefers directly setting NUM_THREADS variable in the env. Can we set `NUM_THREADS` here only if it is not set? Thanks!

Reply:
Fixed, thanks.

Reply:
Fixed, thanks.